src/js/components/Grommet/doc.js

```diff
@@ -23,6 +23,9 @@ export const doc = Grommet => {
     theme: PropTypes.object.description(
       'Custom styles for Grommet app component.',
     ),
+    ssrHeaders: PropTypes.object.description(
```

Comment:
Would there be any downside to always doing this by default and not having the caller have to enable it?

Reply:
I think we have to at least receive what is the user agent. `<Grommet userAgent="blah" />`

Thoughts?

Reply:
To carry this further, what if we had `initialResponsive` and let the caller handle the user-agent stuff, since it is so hard to keep up with?

Reply:
The proposed changes look good, however imo handling of the user agent should be in grommet - my thinking was that responsive and theme handling should be the top 2 features of a flexbox ui library like grommet

Reply:
yup I like having the mobile detect logic inside grommet.

Reply:
I am not sure how we can have this always on by default. how is grommet receiving the user-agent?